src/phoenix/server/app.py

```diff
@@ -157,6 +162,7 @@ def create_app(
             ]
         )
         + [
+            Route("/arize_phoenix_version", version),
```

Comment:
```suggestion
            Route("/version", version),
```

Reply:
this can be a false positive if some other server is running on the same port and have the same end point

Reply:
Cool, sg